src/operator/nn/pooling.cc

```diff
@@ -126,11 +131,15 @@ static bool PoolingShape(const nnvm::NodeAttrs &attrs,
       oshape[2] = 1 +
                   (dshape[2] + 2 * param.pad[0] - param.kernel[0]) /
                       param.stride[0];
-    } else {
-      oshape[2] = 1 + static_cast<int>(std::ceil(
+    } else if (param.pooling_convention == pool_enum::kFull) {
+      oshape[2] = 1 + static_cast<int>(ceil(
                           static_cast<float>(dshape[2] + 2 * param.pad[0] -
                                              param.kernel[0]) /
                           param.stride[0]));
+    } else {
+      oshape[2] = static_cast<int>(ceil(
+                          static_cast<float>(dshape[2] + 2 * param.pad[0]) /
```

Comment:
nit: indentation

Reply:
Unable to find the indentation issue. Also is there a similar linting tool for python to ensure I don't miss out on those. Do we use pylint?

Reply:
If pylint does not complain, I guess it's fine. But it seems not following the Google python style guide: https://github.com/google/styleguide/blob/gh-pages/pyguide.md